src/auto-evo/EvolutionaryTree.cs

```diff
@@ -0,0 +1,229 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using AutoEvo;
+using Godot;
+using Godot.Collections;
```

Comment:
Isn't that only if the Array name would conflict? I think here is the system collections is not imported, there's no name conflict so that kind of using statement is not needed. Well I let Rider handle that but that might be the case why that code is not generated here.

Reply:
I see, but a couple lines below there is a redundant qualifier for some reason.

e.g:
`private readonly System.Collections.Generic.Dictionary<uint, string> speciesNames = new();`

Reply:
I think Rider picks the style that results in the least amount of long qualifiers. So if Godot.Array is used more than once, it would pick to use the long name for the system collection as that results in overall less characters. At least that's my understanding of the logic.

Reply:
I don't know, but this is what reSharper and CI wants